test/gen_rpc/functional_SUITE.erl

```diff
@@ -108,6 +103,32 @@ call(_Config) ->
     ok = ct:pal("Testing [call]"),
     {_Mega, _Sec, _Micro} = gen_rpc:call(?NODE, os, timestamp).
 
+call_anonymous_function(_Config) ->
```

Comment:
I don't like nitpicking but consistent code formatting is always nice. Would you mind if I reformatted your code after merging the PR to make it more consistent with the rest of the project?